api/src/typedefs.ts

```diff
@@ -94,6 +94,14 @@ export const typeDefs = gql`
     ttl: Int
   }
 
+  type YDocSnapshot {
+    id: String
+    repoId: String
+    createdAt: String
+    message: String
+    yDocBlob: String
```

Comment:
This blob doesn’t need to be sent to the front end.

Reply:
> This blob doesn’t need to be sent to the front end.

SG, I was thinking about the "restore" operation, I am not exactly sure if it's better to pass the `yDocBlob` in a single query.

Reply:
Restore should probably happen in the backend yjs server, and front end will retrieve from yjs. I’ll think about this.